README.md

```diff
@@ -303,8 +303,10 @@ postcss([
 }
 ```
 
+**Note**
+
 ```
-Note: To generate the remaining themes, specify the theme you want to generate using defaultTheme option.
+To generate the remaining themes, specify the theme you want to generate using defaultTheme option.
```

Comment:
> NOTE: To generate the remaining themes, specify the theme you want to generate using defaultTheme option.

Reply:
@hipstersmoothie Ive moved the note up as a header